### PR TITLE
Improve performance sync relations

### PR DESCRIPTION
**Pull request by LePetitTim**

### CODE REVIEW

File: geostore/models.py [hunk 1]
```diff
@@ -223,17 +223,18 @@ def sync_relations(self, layer_relation=None):
         for rel in layer_relations:
             logger.info(f"relation {rel}")
             qs = self.get_computed_relation_qs(rel)
+            id_list = qs.values_list("id", flat=True)
             # find relation to delete (in stored relation but not in qs result)
-            to_delete = self.relations_as_origin.filter(relation=rel).exclude(destination_id__in=qs)
+            to_delete = self.relations_as_origin.filter(relation=rel).exclude(destination_id__in=id_list)
 
-            logger.info(f"{to_delete.count()} element(s) to delete")
+            # logger.info(f"{to_delete.count()} element(s) to delete")
```
Comment (jrmi) [inline geostore/models.py]: Do you need to keep this comments

File: geostore/models.py [hunk 2]
```diff
@@ -223,17 +223,18 @@ def sync_relations(self, layer_relation=None):
...
 
             to_delete.delete()
 
             # find relation to add (not in stored relation but in qs
-            qs = qs.exclude(pk__in=self.relations_as_origin.filter(relation=rel)
-                                                           .values_list('destination_id', flat=True))
-            logger.info(f"{len(qs)} element(s) to add")
+            qs = Feature.objects.filter(id__in=id_list).exclude(pk__in=self.relations_as_origin.filter(relation=rel)
+                                                                .values_list('destination_id', flat=True))
+            # logger.info(f"{len(qs)} element(s) to add")
```
Comment (jrmi) [inline geostore/models.py]: And this one?

**Review by jrmi — CHANGES_REQUESTED** [review]

File: geostore/models.py (hunk 2 above)
Comment (LePetitTim) [reply geostore/models.py]: No

File: geostore/models.py (hunk 1 above)
Comment (LePetitTim) [reply geostore/models.py]: No

**Pull request merged**